tox.ini

```diff
@@ -35,10 +37,18 @@ usedevelop = True
 
 [testenv:metadata-validation]
 deps =
-    collective.checkdocs
-    twine
+  collective.checkdocs >= 0.2
```

Comment:
Plz don't change the indentation.

Reply:
I am afraid you cannot blame me for *FIXING* the identation. That makes the indentation on tox.ini deps consistent with previous deps line. Looks at the file, its a mix of 2 and 4 identations.

Reply:
it's mostly 4 spaces. this PR is not about reformatting things. so yes, you're breaking it now.